packages/core/src/card/InteractableCard.css

```diff
@@ -1,50 +1,85 @@
 /* Styles applied to InteractableCard */
 .saltInteractableCard {
   box-shadow: var(--saltCard-boxShadow, var(--salt-overlayable-shadow));
-  border-width: var(--saltCard-borderWidth, var(--card-borderWidth));
+  border-width: var(--saltCard-borderWidth, var(--salt-size-border));
   border-style: var(--saltCard-borderStyle, var(--salt-container-borderStyle));
   border-color: var(--saltCard-borderColor, var(--salt-accent-borderColor));
   border-radius: var(--saltCard-borderRadius, 0);
   display: block;
+  position: relative;
   transition: box-shadow var(--salt-duration-instant) cubic-bezier(0.4, 0, 0.2, 1);
   padding: var(--saltCard-padding, var(--salt-spacing-300));
+  --card-accent-color: var(--saltCard-accentColor, var(--salt-accent-borderColor));
 }
 
 /* Styles applied to InteractableCard when variant="primary" */
 .saltInteractableCard-primary {
   background: var(--saltCard-background, var(--salt-container-primary-background));
+  border-color: var(--salt-container-primary-borderColor);
 }
 
 /* Styles applied to InteractableCard when variant="secondary" */
 .saltInteractableCard-secondary {
   background: var(--saltCard-background, var(--salt-container-secondary-background));
+  border-color: var(--salt-container-secondary-borderColor);
+}
+
+.saltInteractableCard-small {
+  padding: var(--saltCard-padding, var(--salt-spacing-100));
```

Comment:
To be consistent it's probably better to do the same thing as card-accent-color and have a CSS variable get assigned here and keep the padding application in the main class.